quickstarts/hello-rules/README.md

```diff
@@ -0,0 +1,194 @@
+## Red Hat Process Automation Manager KIE Server decisions Quickstart
+
+Note that, this is the development branch, the target images might not be available here, instead you can look at the [released branch](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.x/quickstarts/hello-rules)
+
+This quickstart is intended to be used with the [RHPAM KIE Server](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/main/kieserver) image.
+
+## How to use it?
+
+The template below will be used for this quickstart:
+
+[rhdm713-prod-immutable-kieserver](https://github.com/jboss-container-images/rhdm-7-openshift-image/blob/main/templates/rhdm713-prod-immutable-kieserver.yaml)
```

Comment:
this references the other repository for https://github.com/jboss-container-images/rhdm-7-openshift-image, is it ok?